src/api/index.js

```diff
@@ -1,2 +1,3 @@
-require('../../babel-transformer');
+require('reify');
```

Comment:
If you really don't to use `--require`, then do not duplicate those requires and import reify-transformer here